src/actor/actor-document.js

```diff
@@ -128,24 +128,17 @@ export class FabulaUltimaActor extends Actor {
 // 		return super.createEmbeddedDocuments(embeddedName, newData, options);
 // 	}
 //
-// 	/**
-// 	 * Override initializing a character to set default portraits.
-// 	 * @param {object} data object of an initialized character.
-// 	 * @param {object?} options optional object of options.
-// 	 */
-// 	static async create(data, options) {
-// 		if (!data.img) {
-// 			switch (data.type) {
-// 				case "party":
-// 					data.img = "systems/fabula-ultima/assets/fbl-sun.webp";
-// 					break;
```

Comment:
What's this party case we're losing?

Reply:
I'm not entirely sure what the two default images look like, but it's probably worth just commenting in that method as is. I was thinking about it and we probably don't want to use an image we (or foundry) don't own 😬